src/models/v2/message-trait.ts

```diff
@@ -18,12 +17,17 @@ import type { SchemaInterface } from '../schema';
 import type { TagsInterface } from '../tags';
 
 import type { v2 } from '../../spec-types';
+import { getDefaultSchemaFormat } from '../../schema-parser';
```

Comment:
Please move that import above the type imports, here and in other files :) Only for consistency in source code.

Reply:
Fixed

Reply:
Pretty sure this is not solved? as I understand what @magicmatatjahu asks for is that `import { getDefaultSchemaFormat } from '../../schema-parser';` should be above line 10, i.e. the `import type { BindingsInterface } from '../bindings';`. right? 🤔

Reply:
exactly, above all type imports